database/tables/load_bond_db.sql

```diff
@@ -0,0 +1,148 @@
+
+-- Create custom types
+
+CREATE TYPE AssetType AS ENUM ('CURRENCY', 'BOND');
+CREATE TYPE QuoteStatus AS ENUM ('OPEN', 'CLOSED');
+CREATE TYPE OrderStatus AS ENUM ('OPEN', 'MATCHED', 'SETTLED');
+CREATE TYPE OrderType AS ENUM ('MARKET', 'LIMIT');
+CREATE TYPE OrderAction AS ENUM ('BUY', 'SELL');
+CREATE TYPE OrganizationType AS ENUM ('TRADING_FIRM', 'PRICING_SOURCE');
+CREATE TYPE Role AS ENUM ('MARKET_MAKER', 'TRADER');
+CREATE TYPE PaymentType AS ENUM ('COUPON', 'REDEMPTION');
+CREATE TYPE CouponType AS ENUM ('FIXED', 'FLOATING');
+CREATE TYPE CouponFrequency AS ENUM ('QUARTERLY', 'MONTHLY', 'DAILY');
+
+
+-- Create tables
+
+CREATE TABLE IF NOT EXISTS bonds (
+  id                        bigserial   PRIMARY KEY,
+  bond_id                   varchar,
+  issuing_organization_id   varchar,
+  amount_outstanding        bigint,
+  coupon_rate               bigint,
+  first_coupon_date         date,
```

Comment:
Dates will be inputted to the REST API (and stored in state) as a timestamp, so it might make sense to store them in the same format here. That way, we can leave it to the client to do any sort of date formatting.